server/routes/testutils/testData.ts

```diff
@@ -212,18 +212,20 @@ export default class TestData {
   }: Partial<RequestSessionTemplateVisitStatsDto> = {}): RequestSessionTemplateVisitStatsDto => ({ visitsFromDate })
 
   static visitStats = ({
-    minimumCapacity = { open: 3, closed: 1 },
-    visitCount = 8,
+    cancelCount = 0,
+    minimumCapacity = { open: 4, closed: 3 },
+    visitCount = 7,
     visitsByDate = [
       {
         visitCounts: {
           open: 4,
-          closed: 4,
+          closed: 3,
```

Comment:
Tweaking test data numbers a bit so properties have different values to avoid potential confusion (and so - hopefully - they add up to realistic totals!)